charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml

```diff
@@ -106,4 +106,16 @@ rules:
   - watch
   - get
   - update
+- apiGroups:
+  - analysisruns
```

Comment:
isn't this apiGroup covered by the one `"*"` below?

Reply:
Yes. Let me remove this